### PR TITLE
fix isvalidpoint

### PR DESCRIPTION
**Pull request by benwulfe**
fixes an exception in is_valid_point.

### CODE REVIEW

**Review by erickzul — CHANGES_REQUESTED** [review]
Per offline discussion, let's have get_data_at_coords call is_valid_point and throw an exception if the result is False.

**Comment by benwulfe** [comment]
should be fixed now

**Comment by erickzul** [comment]
Do we need to add/run any tests for code here or in ddf-isoscaopes that depend on this?

**Review by erickzul — APPROVED** [review]
Approved pending confirming code runs successfully in ddf-isoscapes with these changes.

File: raster.py [hunk 1]
```diff
@@ -259,12 +259,21 @@ def get_data_at_coords(dataset: AmazonGeoTiff, x: float, y: float, month: int) -
...
   if np.ma.is_masked(value):
-    raise ValueError("Coordinates are masked")
+    return None
+  else:
+    return value
+
+def get_data_at_coords(dataset: AmazonGeoTiff, x: float, y: float, month: int) -> float:
+  # x = longitude
+  # y = latitude
+  value = _get_data_at_coords(dataset, x, y, month)
```
[the remark this answers is not in the document]
Comment (benwulfe) [reply raster.py:269]: a naming pattern i have seen in this context is "tryX"
_try_get_value_at_coords....

seems reasonable.  though in general the underscore is usually hidden, so shouldnt be confused, i do agree the name could be clearer.

**Comment by benwulfe** [comment]
I did not see anything impacted, but those could have been in branches.

**Pull request merged**